TagV3.0_U575VGT/Core/Src/Sensor Src/BNO08x.c

```diff
@@ -425,6 +325,53 @@ static HAL_StatusTypeDef IMU_poll_new_data(IMU_HandleTypeDef* imu, uint32_t time
 	}
 
 	return HAL_OK;
-
 }
 
+static void IMU_configure_reports(IMU_HandleTypeDef * imu, uint8_t reportID, bool isLastReport){
+
+	//Need to setup the IMU to send the appropriate data to us. Transmit a "set feature" command to start receiving rotation data.
+	//All non-populated bytes are left as default 0.
+	uint8_t transmitData[256] = {0};
```

Comment:
We know the required size of `transmitData`.
`uint8_t transmitData[IMU_CONFIGURE_REPORT_LENGTH]`

Reply:
should be fixed in newest commit